FEAT-#7139: Use ray-core instead of ray-default (#6955)
modin-project/modin: docs/getting_started/installation.rst

Otherwise, installation of ``modin[mpi]`` may fail. Refer to
`Installing with pip`_ section of the unidist documentation for more details about installation.

> **Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray` instead of `ray[default]`.

Reviewer: ```suggestion
**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: ``ray`` instead of ``ray[default]``.
```

This means that the dashboard and cluster launcher are no longer installed by default.

> If you need those, consider installing `ray[default]` along with `modin[ray]`.

Reviewer: ```suggestion
If you need those, consider installing ``ray[default]`` along with ``modin[ray]``.
```

Otherwise, installation of ``modin[mpi]`` may fail. Refer to
`Installing with pip`_ section of the unidist documentation for more details about installation.

> **Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: ray-core instead of ray-default.

Reviewer: same

> **Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: ray-core instead of ray-default.
> This means that the dashboard and cluster launcher are no longer installed by default.

Author: ```suggestion
**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray` instead of `ray[default]`.
This means that the dashboard and cluster launcher are no longer installed by default.
If you need those, consider installing `ray[default]` (through pip) along with `modin[ray]` or `ray-default` (through conda).
```

| modin-all                       | Dask, Ray, Unidist, HDK   |          Linux              |
+---------------------------------+---------------------------+-----------------------------+

> **Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray-core` instead of `ray-default`.

Reviewer: ```suggestion
**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: ``ray-core`` instead of ``ray-default``.
```

This means that the dashboard and cluster launcher are no longer installed by default.

> If you need those, consider installing `ray-default` along with `modin-ray`.

Reviewer: ```suggestion
If you need those, consider installing ``ray-default`` along with ``modin-ray``.
```
